packages/lodestar/src/util/bit.ts

```diff
@@ -0,0 +1,15 @@
+import {BitArray} from "@chainsafe/bit-utils/lib/base";
+
+/**
+ * Returns count of true bits in bitarray
+ * @param bitArray
+ */
+export function getBitCount(bitArray: BitArray): number {
+  let count = 0;
+  for(let i = 0; i < bitArray.bitLength; i++) {
+    if(bitArray.getBit(i)) {
+      count++;
+    }
+  }
+  return count;
+}
```

Comment:
we should maybe put this as a method on BitArray

Reply:
https://github.com/ChainSafe/bit-utils/issues/7

Reply:
https://github.com/ChainSafe/bit-utils/issues/7